<?xml version="1.0"?>
<launch>
    <arg name="show_obstacles" default="true"/>
    <node pkg="collision_ik" type="rviz_viewer.py" name="rviz_viewer" output="screen" args="$(arg show_obstacles)"/>
</launch>
